codec_specs.md

```diff
@@ -944,6 +944,14 @@ Codec Name: ARIB STD-B24 subtitles
 Description: This is the textual subtitle format used in the ISDB/ARIB broadcasting standard.
 For more information, see (#arib-isdb-subtitles) on ARIB (ISDB) subtitles.
 
+### S_C608
+
+Codec ID: S_C608
+
+Codec Name: CEA-608 Captions
+
+Description: Each Block of a S_C608 subtitle track stores the c608 data as a array of one or more octet pairs from one data channel of a CEA-608 data stream with each octet pair corresponding to a video frame. The "LABEL" Matroska tag is RECOMMENDED for storage the channel number of the CEA-608 (such as "CC1", "CC2", "CC3" or "CC4"). For more information about this format see [@!ANSI-CTA-608-E-S-2019].
```

Comment:
CC1 & CC2 share field 1 and can not be separated.
CC3 & CC4 share field 2 and can not be separated.

IMO it would be better to indicate the field, or even better to say that for frames there should be 4 bytes (2 bytes of field 1 then 2 bytes of field 2) so we avoid a have to manage tags.